server/models/users.ts

```diff
@@ -0,0 +1,37 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../database/connection';
+
+const Users = sequelize.define('User', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  mobile: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  img: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  location: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  role: {
+    type: DataTypes.STRING,
+    allowNull: false,
```

Comment:
For the role search about `enum` type, since we have specific roles to choose from.
Another thing is that the student will not have a mobile or location so update the `allowNull`.
The last thing is to add a default value to all your images.